downsize_multipage-pdf.bash

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+read -p '1/3: Have you spec.d multipage pdf as Argument + Processfile alone in proc-dir? Otherwise ^C to abort: '
+read -p '2/3: +wdir contains an "orig/"-directory?: '
+read -p '3/3: And specified TargetDir in script-source? Otherwise ^C to abort: '
+
+echo "saving origfilename"
+origfilename=`basename *.pdf .pdf`
+
+
+read -p 'Check pdf for num. of pages (rename-seqprop hardcoded to 3). ^C and change if not correct: '
+evince ${origfilename}.pdf
+clear
+read -p 'Continue/Start processing?: '
+
+cp ${origfilename}.pdf orig/
+
+echo "splitting pdf into pages"
+pdfseparate $1 %d.pdf
+
+#=comment out for small 1-9 page doks (=eg. like 'svgrefx1-2021' etc) (**!)
+echo "(renaming sequential pdf:s to 'sequential proper')"
+~/dropbox-legacy/scripts/rename-sequentialproper.bash
```

Comment:
Might be good to generalize, assume that rename-sequentialproper.bash is in the path rather than use a local (non-portable) path.